lib/msal-browser/src/interaction_handler/PopupHandler.ts

```diff
@@ -119,7 +119,7 @@ export class PopupHandler extends InteractionHandler {
             if (popup) {
                 popupWindow = popup;
                 popupWindow.location.assign(urlNavigate);
-            } else if (typeof popup === "undefined") { 
+            } else if (typeof popup === "undefined") {
```

Comment:
Is undefined supposed to be a string here?

Reply:
Nope, this logic is setup to handle `popup` being a Window, null, or undefined.

Reply:
Think so. Since this is unrelated, can we defer this to a new PR in case we plan to change it?

Reply:
Fine by me